### ISSUE TITLE
when set params with array obect, i want test=1&test=2, but return test[]=1&test[]=2

### ISSUE THREAD

**Issue by KerwinChung** [pagekit/vue-resource#615]
$http.get(url, { 
     params: { 
         test :[1,2]
    }
}
i want  url?test=1&test=2
but return test[]=1&test[]=2

can you fix it ?
./src/url/index.js

function serialize(params, obj, scope) {
.......

        if (scope) {
            key = scope + '[' + (plain || hash ? key : '') + ']';
        }
....
}

**Comment by oppianmatt** [comment]
yes this would be good for other backends that don't use php style post variables for arrays

**Comment by flyinglove** [comment]
is this modification released?

**Comment by KerwinChung** [comment]
I don’t know. The web development is so far ago, now I develop with python.

在 2018年5月29日，下午12:00，flyinglove <notifications@github.com<mailto:notifications@github.com>> 写道：


is this modification released?
